src/main/java/com/microfocus/application/automation/tools/octane/ImpersonationUtil.java

```diff
@@ -0,0 +1,45 @@
+package com.microfocus.application.automation.tools.octane;
+
+import com.hp.octane.integrations.exceptions.PermissionException;
+import com.microfocus.application.automation.tools.model.OctaneServerSettingsModel;
+import com.microfocus.application.automation.tools.octane.configuration.ConfigurationService;
+import hudson.model.User;
+import hudson.security.ACL;
+import hudson.security.ACLContext;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.Collections;
+
+public class ImpersonationUtil {
```

Comment:
Please add a description, on the purpose of this new class.

Reply:
Added